### ISSUE TITLE
error in pspnet.prototxt

### ISSUE THREAD

**Issue by jeansely** [Vladkryvoruchko/PSPNet-Keras-tensorflow#8]
Hello,
when run weight-converter this error occur:

 "caffe.LayerParameter" has no field named "bn_param".

                          i replace it with "batch_norm_param"
again occur below error:
 "caffe.BatchNormParameter" has no field named "slope_filler".

**Comment by Vladkryvoruchko** [comment]
Hello @jeansely . There is modified prototxt in /utils/model

**Comment by jeansely** [comment]
now in /utils/model the prototxt is ;
bn_param {
    slope_filler {
      type: "constant"
      value: 1
    }
    bias_filler {
      type: "constant"
      value: 0
    }
    frozen: true
    momentum: 0.95

what are  replace these lines ?

**Comment by hujh14** [comment]
It is from here.
https://github.com/hszhao/PSPNet/blob/master/evaluation/prototxt/pspnet50_ADE20K_473.prototxt

**Comment by jeansely** [comment]
when run weigth -converter occur error in prototxt "line 109" 


 has no field named "bn_param".
and 
"caffe.BatchNormParameter" has no field named "slope_filler".

**Comment by CuriousCat-7** [comment]
I hold the same problem

**Comment by tangsanli5201** [comment]
I hold the same problem too.

**Comment by Vladkryvoruchko** [comment]
@CuriousCat-7 @tangsanli5201  Please use already converted h5, json weights. Links that can be found in README.md 
I just cannot remember what was done with this converter, so if I find out i will leave a comment in this thread

**Comment by tangsanli5201** [comment]
@Vladkryvoruchko Yeah. I tried to use the already converted json weights, but it raised an error: 'bad marshal data...'., which I thought it is caused by the different version of Keras( mine is 2.1.2). And it's solved now: I generated the json file through Keras 2.1.2 and replaced the old one. It's working. Thanks!

**Comment by RahavLussato** [comment]
@tangsanli5201 how did you manage to generate the json ? i have the same `bad marshal data...` issue
